plugins/kubernetes/app/models/kubernetes/role_validator.rb

```diff
@@ -241,6 +242,20 @@ def validate_stateful_set_service_consistent
       @errors << "Service metadata.name and StatefulSet spec.serviceName must be consistent"
     end
 
+    def validate_daemon_set_supported
+      return unless daemon_set = @elements.detect { |t| t[:kind] == "DaemonSet" }
+
+      if daemon_set.dig(:apiVersion) != "apps/v1"
```

Comment:
This is not required, by the way, support for apps/v1 is different issue. But leave it here, anyway.

Reply:
extensions/v1beta1 which we used before does not do any updates to the pod when used, so we have to force apps/v1